src/stories/containers/Finances/components/OverviewCardKeyDetailsBudget/DoughnutChartFinances/DoughnutChartFinances.tsx

```diff
@@ -0,0 +1,196 @@
+import styled from '@emotion/styled';
+import { useThemeContext } from '@ses/core/context/ThemeContext';
+
+import lightTheme from '@ses/styles/theme/light';
+import ReactECharts from 'echarts-for-react';
+import React from 'react';
+import type { DoughnutSeries } from '@ses/core/models/interfaces/doughnutSeries';
+
+interface Props {
+  doughnutSeriesData: DoughnutSeries[];
+}
+
+const DoughnutChartFinances: React.FC<Props> = ({ doughnutSeriesData }) => {
+  const { isLight } = useThemeContext();
+
+  const options2 = {
+    color: doughnutSeriesData.map((data) => data.color),
+    tooltip: {
+      show: true,
+      trigger: 'item',
+      label: {
+        show: false,
+        position: 'center',
+      },
+      emphasis: {
+        width: 40,
+      },
+
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      formatter: function (params: any) {
```

Comment:
could we replace the `any` type here for the correct echart typing? Take a look here: 
- Nice Article: https://juejin.cn/post/7066817168335175688
- https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/echarts/options/tooltip.d.ts#L390
- https://stackoverflow.com/questions/66992620/echarts-tooltip-typescript
- https://github.com/apache/echarts/issues/14277